### PR TITLE
*: clean up MvccStore in mocktikv

### PR DESCRIPTION
**Pull request by tiancaiamao**
MVCCLevelDB and MvccStore both implement MVCCStore interface, we can
retain just MVCCLevelDB.

@coocood

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: server/http_handler_test.go [hunk 1]
```diff
@@ -221,7 +221,8 @@ func (ts *HTTPHandlerTestSuite) TestRegionsFromMeta(c *C) {
 }
 
 func (ts *HTTPHandlerTestSuite) startServer(c *C) {
-	mvccStore := mocktikv.NewMvccStore()
+	mvccStore, err := mocktikv.NewMVCCLevelDB("")
```
Comment (ngaut) [inline server/http_handler_test.go]: DRY, could you create a a function like MustNewStore and check err in the function?

**Pull request merged**